### ISSUE TITLE
decontam use

### ISSUE THREAD

**Issue by Zarzarn95** [benjjneb/decontam#147]
Hi,
 can we use Decontam package for whole genome microbiome data? how?

my data doesn't categorize in OTUs or ASVs.

**Comment by benjjneb** [comment]
Yes. You would just use the table of taxonomic abundances from a metagenomic profiling software, e.g. a species table with associated abundances per sample.

**Comment by Zarzarn95** [comment]
it's the error I'm receiving while using "decontam" package. what should I do now? (my file is excel not a gz or zip file)

> ps <- readRDS(system.file("data", "data2", package="decontam"))
Error in gzfile(file, "rb") : cannot open the connection
In addition: Warning message:
In gzfile(file, "rb") :
  cannot open compressed file '', probable reason 'Invalid argument'

**Comment by benjjneb** [comment]
The command that is causing this error doesn't seem to make sense.

What are you trying to accomplish with the `readRDS` command?
Do you have access to any folks at your place of work that could help with R language?
